judo-ui-react/src/main/resources/actor/src/pages/actions/ExportAction.fragment.hbs

```diff
@@ -5,8 +5,23 @@ const {{ simpleActionDefinitionName action.actionDefinition }} = async (queryCus
 {{/ if }}
   try {
     setIsLoading(true);
-    console.log(queryCustomizer);
-    alert('Not yet implemented');
+{{# if container.isRelationSelector }}
+    const response = await {{ getServiceImplForPage page }}.export(undefined, queryCustomizer);
+{{ else }}
+  const response = await {{ getServiceImplForPage page }}.export{{# if action.targetDataElement }}{{ firstToUpper action.targetDataElement.name }}{{/ if }}({{# if (pageHasSignedId page) }}{ __signedIdentifier: signedIdentifier } as JudoIdentifiable<any>{{ else }}undefined{{/ if }}, queryCustomizer);
+{{/ if }}
+
+    if (response && response.status === 200) {
+      const fileName = response.headers['content-disposition'].match(/filename=\"(.*)\"/)[1];
```

Comment:
We should export this functionality as a helper function. If it was copied from somewhere else, use the helper there as well.

Reply:
[ccea228](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/318/commits/ccea228dc8339b1824f3087acce13983a79a8218)